README.md

```diff
@@ -51,26 +51,26 @@
 ![Alt text](docs/images/vulnerable-eb-cnames.png?raw=true "Detect vulnerable ElasticBeanstalk CNAMEs")
 
 ## documentation
-[Manual scans - AWS](manual_scans/aws/README.md)  
```

Comment:
the double spaces at the end of each line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

Reply:
so I think this is more of a problem with how markdown was formatted. If we want to have new lines, we should do bullet list

Reply:
My intention was to make it look more like a table of contents, hence not using bullet points. 

However after some research and testing, I discovered that the double spaces can be replaced by `<br>` which forces a line break and still passes the pre-commit check.

I'd suggest using that in the relevant places, which then means there should be no need to exclude these `.MD` / `.md` files from the pre-commit check.